app/forms/consent_form.rb

```diff
@@ -2,9 +2,6 @@ class ConsentForm < QuestionsForm
   include BirthDateHelper
   set_attributes_for(
     :intake,
-    :primary_consented_to_service,
```

Comment:
Consenting has been removed into the after update steps so that we don't end up consenting clients that we're offboarding.